src/TodoList.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+
+// function to list from todos from array
+function TodoList(){
+    const todoList = [
+        {
+          'id':1,
+          'title':'waitting assignment'
+        }, {
+          'id':2,
+          'title':'ongoing assignment'
+        }, {
+          'id':3,
+          'title':'Complete assignment'
+        },
+      ];
+    return (
+        <>
+          <ul>
+            {todoList.map(function(item){
+                return <li key={item.id}>{item.title}</li>
+              })}
+          </ul>
+        </>
```

Comment:
Since <ul> is a single element already, you do not need to wrap it in a document fragment. No need to make a change and resubmit- I just wanted to make you aware of that detail.